src/Tracy/assets/BlueScreen/content.phtml

```diff
@@ -193,10 +193,11 @@ $code = $exception->getCode() ? ' #' . $exception->getCode() : '';
 		<?php $bottomPanels = [] ?>
 		<?php foreach ($this->renderPanels(NULL) as $panel): ?>
 		<?php if (!empty($panel->bottom)) { $bottomPanels[] = $panel; continue; } ?>
+		<?php $collapsedClass = (isset($panel->collapsed) ? $panel->collapsed : TRUE) ? ' tracy-collapsed' : ''; ?>
```

Comment:
Can be simplified to 

~~~php
$collapsedClass = !isset($panel->collapsed) || $panel->collapsed ? ' tracy-collapsed' : '';
~~~